biotracks/cmso.py

```diff
@@ -39,3 +39,6 @@
 OBJECT_ID = "cmso_object_id"
 LINK_ID = "cmso_link_id"
 TRACK_ID = "cmso_track_id"
+
+SPATIAL_UNIT = "cmso_spatial_unit"
```

Comment:
Really a detail, but shall we say '`SPATIAL_UNIT`' and '`TEMPORAL_UNIT`', or
'`SPACE_UNIT`' and '`TIME_UNIT`'? Simply for consistency. Feel free to ignore.